docs/Examples.rst

```diff
@@ -0,0 +1,17 @@
+========
+Examples
+========
+
+Dedupe is a library and not a stand-alone command line tool. To
+demonstrate its usage, we have come up with a `few example recipes for
+different sized datasets for you
+<https://github.com/dedupeio/dedupe-examples/archive/0.5.zip>`__
+(`repo <https://github.com/dedupeio/dedupe-examples>`__, as well as
```

Comment:
the "repo" is rendering strangely, it may no be doing what you intend?

Reply:
![Screenshot 2022-02-09 at 15-07-22 Examples — dedupe 2 0 11 documentation](https://user-images.githubusercontent.com/536941/153281737-d768280e-424c-4d04-b0af-ebd75385ab14.png)